hugo/data/sref_locations.yaml

```diff
@@ -15,6 +15,8 @@ vm_interpreter: [systems, filecoin_vm, interpreter]
 ### blockchain
 block_producer: [systems, filecoin_blockchain, struct]
 chainsync: [systems, filecoin_blockchain, chainsync]
+block: [systems, filecoin_blockchain, block]
+block_validation: [systems, filecoin_blockchain, block, block_validation]
```

Comment:
ACTION REQUIRED: the link to block validation is dead and I can't figure out how to make it redirect to the correct place.  I need help if we want this to merge with working links.

Reply:
tried and failed to help. Keep as is, we'll need to do an overall pass on all links eventually anyways.